core/commands/stat.go

```diff
@@ -137,49 +130,44 @@ Example:
 				res.Emit(&totals)
 			}
 			if !doPoll {
-				return
+				return nil
 			}
 			select {
 			case <-time.After(interval):
 			case <-req.Context.Done():
-				return
+				break
```

Comment:
`return req.Context.Err()`?

Reply:
That would return an error and non-zero exit code to the user, even though everything was in order. Do you think this is a good idea?

Reply:
Ah, I see. You're right.